### ISSUE TITLE
high cpu load when move mouse over textlines

### ISSUE THREAD

**Issue by ooab** [LiMinggang/madedit-mod#265]
**Please provide the following information**

Madedit-Mod version (or branch): 0.4.11, 0.4.12

platform/architecture: windows7 64bit

compiler and compiler version: 

please describe what symptom you see, what you would expect to see instead and how to reproduce it.

Open a text file, move mouse over textlines continuously, cpu(single core) percentage increases to 25% rapidly. At the same time, the curor show Beam and Arrow alternately.

two PC tested:
1. Thinkpad T470, windows7 64bit, high cpu load
2.  Old desktop PC, windows7 64bit, no problem

when comment out this line:
void MadEdit::OnMouseMotion( wxMouseEvent &evt ) 
{
    if( m_MouseInWindow ) 
    {
        **//UpdateCursor( evt.m_x, evt.m_y );** 
    }
}
cpu load dropped back to a normal level on both computers.  
compare to code of MadEdit 0.29,  
new versions comment out several lines in function UpdateCursor()
wxSetCursor( XXXCursor );
**SetCursor(XXXCursor);** 

if I restore the calls to SetCursor(XXXCursor), my builds works on both computers.

**Comment by LiMinggang** [comment]
Di you mean replace wxSetCursor with SetCursor would fix the issue?

**Comment by LiMinggang** [comment]
You can fire a pull request if you interested in improving this project. Or I can fix it in next release.

**Comment by LiMinggang** [comment]
Would you please try the latest code to see if it fixed your problem.

**Comment by ooab** [comment]
not replace.  my fix have both calls, for example:
if( m_DragCopyFlag )
{
    wxSetCursor( DragCopyCursor );
    SetCursor(DragCopyCursor);
}
I am new to wxWidgets,  and i am confused by the code, what is the difference between wxSetCursor and SetCursor?
any way, I will try your fix code.

**Comment by LiMinggang** [comment]
wxSetCursor is to set the cursor globally(for all windows if you are not switch to other windows and only move your mouse over the windows displayed on your desktop). SetCursor is to change the cursor for the window/control itself.

No problem. Thanks for your finding. This is the bug I introduced while implementing the Dnd edit for MadEdit.

I've applied the change as you did to MadEdit-Mod. Don't worry.

Thanks!
